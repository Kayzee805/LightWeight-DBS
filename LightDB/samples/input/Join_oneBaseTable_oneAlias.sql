Select * FROM Sailors S, Enrollments E WHERE E.student_id < S.C AND E.course_id < S.C;
