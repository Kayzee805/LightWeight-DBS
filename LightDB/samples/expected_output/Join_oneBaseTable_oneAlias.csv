5,100,500,100,301
5,100,500,104,304
5,100,500,104,305
5,100,500,102,202
5,100,500,101,202
5,100,500,102,302
5,100,500,101,301
5,100,500,103,305
5,100,500,103,303
6,300,400,100,301
6,300,400,104,304
6,300,400,104,305
6,300,400,102,202
6,300,400,101,202
6,300,400,102,302
6,300,400,101,301
6,300,400,103,305
6,300,400,103,303

